### PR TITLE
feat: refactor to move so library into python source dir

### LINKED ISSUE #1114: Refactor openmldb python sdk and do not move python files to package
Now we move the python source files into `build` to package. We should refactor the cmakelists and make python source code managed easily in source directory.

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Resolve https://github.com/4paradigm/OpenMLDB/issues/1114 .

Now we can use `python ./setup.py develop` to install openmldb in dev environment and do not need to build whl package for testing.

### CODE REVIEW

**Comment by tobegit3hub** [comment]
There are some modules after refactor.

image


Here is the new docs to setup python dev env.

image

File: python/openmldb_native/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```
Comment (dl239) [inline python/openmldb_native/__init__.py:1]: What is the purpose of this file

**Review by dl239 — APPROVED** [review]

File: python/openmldb_dbapi/dbapi.py [hunk 2]
```diff
@@ -233,43 +237,65 @@ def execute(self, operation, parameters=()):
...
                             continue
-                    colType = schema.GetColumnType(idx)
-                    if colType != driver.sql_router_sdk.kTypeString:
-                        continue
-                    if isinstance(parameters[name], str):
-                        strSize += len(parameters[name])
+                        if isinstance(parameters[name], str):
+                            strSize += len(parameters[name])
+                        else:
+                            raise DatabaseError("{} value tpye is not str".format(name))
```
Comment (HuilinWu2) [inline python/openmldb_dbapi/dbapi.py]: tpye -> type

File: python/sqlalchemy_openmldb/openmldb_dialet.py [hunk 3]
```diff
@@ -86,9 +89,9 @@ def __init__(self, dialect):
     def format_drill_table(self, schema, isFile=True):
         pass
 
-class OpenMLDBDialect(default.DefaultDialect):
+class OpenmldbDialect(default.DefaultDialect):
     
-    name = "openmldbapi"
+    name = "openmldb_dialet"
```
Comment (HuilinWu2) [inline python/sqlalchemy_openmldb/openmldb_dialet.py]: openmldb_dialet -> openmldb_dialect
also, the filename should be `openmldb_dialect.py`

**Review by HuilinWu2 — APPROVED** [review]
some misspell

**Review by HuilinWu2 — APPROVED** [review]
LGTM

**Review by HuilinWu2 — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**